package/MDAnalysis/analysis/base.py

```diff
@@ -450,6 +487,48 @@ def run(self, start=None, stop=None, step=None, frames=None,
         self._conclude()
         return self
 
+    def _setup_bslices(self, start=None, stop=None, step=None, frames=None):
+        self._bslices = [(start, stop, step, frames)]
+    
+    def _setup_scheduler(self, **kwargs):
+        self._scheduler_kwargs = kwargs
+
+    def run(self, start=None, stop=None, step=None, frames=None,
+            verbose=None, scheduler=None, 
+            *, progressbar_kwargs={},
+            ):
+        if scheduler is None: # fallback to the local scheduler
+            self._compute(start=start, stop=stop, step=step, frames=frames, 
+            verbose=verbose, progressbar_kwargs=progressbar_kwargs)
+        else:
```

Comment:
Raise an error when dask is using threads scheduler.

Reply:
I'm currently explicitly passing `processes` to dask, so I guess it's ok